server/task_executor.go

```diff
@@ -169,6 +169,21 @@ func postMigration(ctx context.Context, server *Server, task *api.Task, vcsPushE
 		}
 	}
 
+	// After establishing a new baseline, we should remove schema drift anomalies.
+	if mi.Type == db.Baseline {
```

Comment:
I think any type of migration will fix the schema drifts. @tianzhou to confirm.

Reply:
https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/server/anomaly_scanner.go?L308

We can remove the if-wrapper.